src/command_modules/azure-cli-vm/setup.py

```diff
@@ -34,7 +34,7 @@
 DEPENDENCIES = [
     'azure-mgmt-msi==0.2.0',
     'azure-mgmt-authorization==0.50.0',
-    'azure-mgmt-compute==4.6.1',
+    'azure-mgmt-compute==4.6.2',
```

Comment:
you might need to update  other command  modules such as acs

Reply:
Yes @yugangw-msft acs and servicefabric have been updated.